Assets/Scripts/AndroidPermissionsManager.cs

```diff
@@ -1,19 +1,42 @@
+using System;
 using UnityEngine;
 
 public class AndroidPermissionCallback : AndroidJavaProxy
 {
-    public AndroidPermissionCallback() : base("com.unity3d.player.UnityAndroidPermissions$IPermissionRequestResult") { }
+	private event Action<string> OnPermissionGrantedAction;
```

Comment:
Unfortunatly work policy are tabs, everything setup to be with tabs, I tried to find those settings in VS, but VS settings are nightmare, so it's all tabs as before, hope you will accept it with tabs

All other misspells which I found and you found were fixed